dashboards/jvm/metadata.yaml

```diff
@@ -7,3 +7,11 @@ sample_dashboards:
     related_integrations:
       - id: jvm
         platform: GCE
+  -
+    category: JVM
+    id: jvm-prometheus-overview
+    display_name: JVM Prometheus Overview
+    description: "This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Classes Loaded, Threads, Garbage Collection Count, Garbage Collection Time, Heap Memory, and Non-Heap Memory."
```

Comment:
This description should match the dashboard (e.g. we don't have Heap Memory and Non-Heap Memory, but instead Memory Committed and Memory Used)?